### PR TITLE
Add new pages for release notes

### PR DESCRIPTION
**Pull request by mhegazy**
Mostly copied from https://github.com/Microsoft/TypeScript/wiki/What%27s-new-in-TypeScript, jsut minor changes to make the linter happy.

### CODE REVIEW

**Comment by DanielRosenwasser** [comment]
This should actually go into the website itself, not the Handbook.

**Comment by mhegazy** [comment]
why?

**Comment by mhegazy** [comment]
i can, but why?

File: pages/Releases/TypeScript 1.5.md [hunk 1]
```diff
@@ -0,0 +1,379 @@
...
+
+```ts
+export default class Greeter {
+    sayHello() {
+        console.log("Greetings!");
+    }
+}
+```
+
+Which in tern can be imported using default imports:
```
Comment (sandersn) [inline pages/Releases/TypeScript 1.5.md]: tern -> turn

File: pages/Releases/TypeScript 1.7.md [hunk 2]
```diff
@@ -0,0 +1,210 @@
...
+function f1({ x = 0, y = 0 } = {}) { }
+
+// And can be called as:
+f1();
+f1({});
+f1({ x: 1 });
+f1({ y: 1 });
+f1({ x: 1, y: 1 });
+
+// Type of f2 is (arg?: (x: number, y?: number) => void
```
Comment (sandersn) [inline pages/Releases/TypeScript 1.7.md]: Missing some braces here:

```ts
(arg?: {x: number, y?: number}) => void
```

**Comment by sandersn** [comment]
:+1:

**Pull request closed without merging**